packages/devtools_shared/lib/src/directory_picker/directory_picker.dart

```diff
@@ -0,0 +1,64 @@
+// Copyright 2023 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:io';
+
+import 'package:file/local.dart';
+import 'package:meta/meta.dart';
+import 'package:path/path.dart' as path;
+
+import 'scripts.dart';
+
+class DirectoryPickerManager {
```

Comment:
@kenzieschmoll 
This is my attempt to launch directory picker from the devtool server side. Right now only work on macOS. This also assume the devtool web app and devtool server is on the same machine.